src/main/java/adris/altoclef/tasks/speedrun/BeatMinecraft2Task.java

```diff
@@ -473,14 +480,25 @@ private int getFilledPortalFrames(AltoClef mod, BlockPos endPortalCenter) {
             // If EVERY portal frame is loaded, consider updating our cached filled portal count.
             if (frameBlocks.stream().allMatch(blockPos -> mod.getChunkTracker().isChunkLoaded(blockPos))) {
                 _cachedFilledPortalFrames = frameBlocks.stream().reduce(0, (count, blockPos) ->
-                        count + (isEndPortalFrameFilled(mod, blockPos) ? 1 : 0),
+                                count + (isEndPortalFrameFilled(mod, blockPos) ? 1 : 0),
                         Integer::sum);
             }
             return _cachedFilledPortalFrames;
         }
         return 0;
     }
 
+    private Optional<BlockPos> getADesertTemple(AltoClef mod) { // Stolen? NAHHHHHHHHHH
```

Comment:
Duplicate code 🙃 I'd make a function in `WorldHelper` that passes in a block tracker and runs this

Reply:
Alright